mailpoet/lib/Migrator/AppMigrationTemplate.php

```diff
@@ -0,0 +1,22 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Migrations;
+
+use MailPoet\Migrator\AppMigration;
+
+class AppMigrationTemplate extends AppMigration {
+  public function run(): void {
+    /*
+     * TODO: Implement the migration logic here and remove this comment.
+     *
+     * App Level migration are intended for data migrations that use application level services.
+     * The application level services require the DB structure to be up to date so they run after all DB migrations.
+     *
+     * Do not make changes in the DB structure in App Level migrations!
+     *
+     * You can use:
+     *   $this->connection      For SQL queries using Doctrine DBAL.
```

Comment:
The connection doesn't seem to be used in the template. It's the container instead, so I guess we need to update the comment.